schedule/supportserver_generator.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: create_hdd_gnome
+description: |
+  image creation job used as parent for other jobs testing based on existing installation.
+  To be used as `START_AFTER_TEST=create_hdd_gnome
+vars:
+  DESKTOP: gnome
+  HDDSIZEGB: '30'
+  INSTALLONLY: '1'
+  PUBLISH_HDD_1: SLES-%VERSION%-%ARCH%-Build209.2@%MACHINE%-gnome.qcow2
+  PUBLISH_PFLASH_VARS: SLES-%VERSION%-%ARCH%-Build209.2@%MACHINE%-gnome-uefi-vars.qcow2
+  SEPARATE_HOME: '0'
```

Comment:
Such openQA variables like `HDDSIZE`, `PUBLISH_HDD_1`, `PUBLISH_PFLASH_VARS` are related to environment and will not work when used in schedule. Could you please move them in Job Group Settings.

`INSTALLONLY` can be removed, as it is used in main.pm and as soon we are using schedule here, will be ignored.